assemblyline_core/scaler/controllers/kubernetes_ctl.py

```diff
@@ -1142,7 +1142,7 @@ def start_stateful_container(self, service_name: str, container_name: str,
             service = self.api.read_namespaced_service(deployment_name, self.namespace)
             service.metadata.labels = labels
             service.spec.selector = labels
-            service.spec.ports = [V1ServicePort(port=int(_p)) for _p in spec.container.ports]
+            service.spec.ports = [V1ServicePort(port=int(_p), name=f"port_{_p}") for _p in spec.container.ports]
```

Comment:
is `_p` and object where we could have done a `_p.name if _p.name else f"{port_{int(_p)}"` or `.get("name", f"{port_{int(_p)}")`?
We may be best to use `int()` on `_p` in case it has a weird repr/str and `f"port_{_p}"` is not a simple int.

Reply:
In this case `_p` is a string, we cast it as an integer for the `port` param as that's the requirement for the constructor.